### PR TITLE
[FIX](jsonb)fix jsonb is not in predict column

### PR DESCRIPTION
**Pull request by amorynan**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Review by yiguolei — APPROVED** [review]
LGTM

File: be/src/olap/schema.cpp [hunk 1]
```diff
@@ -180,6 +180,7 @@ vectorized::IColumn::MutablePtr Schema::get_predicate_column_ptr(const Field& fi
         break;
     case FieldType::OLAP_FIELD_TYPE_VARCHAR:
     case FieldType::OLAP_FIELD_TYPE_STRING:
+    case FieldType::OLAP_FIELD_TYPE_JSONB:
```
Comment (airborne12) [inline be/src/olap/schema.cpp:183]: Is it practical to use dictionary encoding for JSONB?

**Comment by amorynan** [comment]
run buildall

**Review by xiaokang — APPROVED** [review]
LGTM

**Comment by amorynan** [comment]
run p0

**Review by yiguolei — APPROVED** [review]
LGTM

**Pull request merged**